### PR TITLE
Add a server that serves content using correct locale

### PR DESCRIPTION
**Pull request by kuguarpwnz**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  // Priority serve any static files.
+  app.use(express.static(path.resolve(__dirname, '../react-ui/build'), { index: false }));
+
+  // All remaining requests return the React app, so it can handle routing.
+  app.get('*', (req, res) => {
+    const language = acceptLanguage.get(req.get('Accept-Language'));
+    const translation = languages[language];
+
+    const content = Mustache.render(template, translation);
```
Comment (MaksymL) [inline server/index.js:51]: do we really need template engine?

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+const numCPUs = os.cpus().length;
+const isDev = process.env.NODE_ENV !== 'production';
+const PORT = process.env.PORT || 3000;
+
+const templatePath = path.join(__dirname, '../react-ui/build', 'index.html');
+const template = fs.readFileSync(templatePath, 'utf-8');
+
+// Multi-process to utilize all CPU cores.
+if (!isDev && cluster.isMaster) {
```
Comment (MaksymL) [inline server/index.js:22]: already optimizing performance? :D

**Review by MaksymL — APPROVED** [review]

File: server/index.js (hunk 1 above)
Comment (kuguarpwnz) [reply server/index.js:51]: we can probably use `template.replace()`. mustache is a cleaner solution which doesn't add any overhead

File: server/index.js (hunk 2 above)
Comment (kuguarpwnz) [reply server/index.js:22]: copy-pasted this code

**Pull request merged**